modules/platforms/dotnet/Apache.Ignite.Core/Client/Cache/ICacheClient.cs

```diff
@@ -422,5 +423,16 @@ public interface ICacheClient<TK, TV>
         /// <typeparam name="TV1">Value type in binary mode.</typeparam>
         /// <returns>Cache instance with binary mode enabled.</returns>
         ICacheClient<TK1, TV1> WithKeepBinary<TK1, TV1>();
+
+        /// <summary>
+        /// Returns cache with the specified expired policy set. This policy will be used for each operation
+        /// invoked on the returned cache.
+        /// <para />
+        /// Expiry durations for each operation are calculated only once and then used as constants. Please
```

Comment:
This is not correct, we actually call `IExpiryPolicy` members on every operation (in contrast with thick mode)